framer/Importer.coffee

```diff
@@ -70,13 +70,16 @@ class exports.Importer
 		# Most layers will have an image, add that here
 		if info.image
 			layerInfo.frame = info.image.frame
-			layerInfo.image = Utils.pathJoin @path, info.image.path
-			
+			if info.image.path.indexOf('data:image') != -1
```

Comment:
I think it's a bit more readable/correct to use https://lodash.com/docs#startsWith here.